README.md

```diff
@@ -12,25 +12,34 @@ Video: https://youtu.be/KiDnPsnLmLU
 
 # Notes/Howto
 
-To run demo with four cones use:
+There has been serious development to extend original code suited mainly for
+modified John Deere X300R garden tractor to more general tool. Now it runs
+on more than four machines, including autonomous boat. It provides logging
+and replay facility, so simple logging of multiple inputs is unified and
+stored in binary format with timestamps.
 
-```
-python ./navpat.py run config/playground15x5.json
-```
+The current code is under `osgar` directory and the remaining code is ported
+now. If you are using OSGAR directly from sources please make sure that
+you setup PYTHONPATH to the root of this project.
+
+The example configurations are stored in `config` folder. JSON files are
+used.
+
+## Examples
+
+### Collect data from sensor(s)
+
+There is a `osgar/robot.py` to run data collection based on given configuration file.
+If you would like to collect GPS data available on serial port use modified version
+of `test-windows-gps.json` for Windows or `test-gps-imu.json` for Linux.
```

Comment:
Can we turn these to links? I am thinking about the user reading the readme and wanting to see what is inside those files. Just make them relative, meaning put osgar/robot.py into both the description and the url here in markdown.